src/test/issues.c2.test.ts

```diff
@@ -91,6 +92,21 @@ describe("Issue Tests", () => {
         );
     });
 
+    it("#2574", () => {
+        const project = convert();
+        const usesDefaultExport = query(project, "usesDefaultExport");
+        const sig = usesDefaultExport.signatures?.[0];
+        ok(sig, "Missing signature for usesDefaultExport");
```

Comment:
There's a helper for this:

```suggestion
        const sig = querySig(project, "usesDefaultExport");
```